catalyst/data/nlp/classify.py

```diff
@@ -0,0 +1,115 @@
+import logging
+from typing import List, Mapping
+
+from transformers import AutoTokenizer
+
+import torch
+from torch.utils.data import Dataset
+
+
+class TextClassificationDataset(Dataset):
```

Comment:
This class is tested in `catalyst/data/nlp/tests/test_text_clf_dataset.py`. Tests can be extended of course.